Libraries/LibWeb/HTML/HTMLFormElement.cpp

```diff
@@ -138,11 +138,19 @@ WebIDL::ExceptionOr<void> HTMLFormElement::submit_form(GC::Ref<HTMLElement> subm
             }
         }
 
-        // FIXME: 4. If the submitter element's no-validate state is false, then interactively validate the constraints
-        //           of form and examine the result. If the result is negative (i.e., the constraint validation concluded
-        //           that there were invalid fields and probably informed the user of this), then:
-        //           1. Set form's firing submission events to false.
-        //           2. Return.
+        // 4. If the submitter element's no-validate state is false, then interactively validate the constraints
+        //    of form and examine the result. If the result is negative (i.e., the constraint validation concluded
+        //    that there were invalid fields and probably informed the user of this), then:
+        //      1. Set form's firing submission events to false.
+        //      2. Return.
```

Comment:
We tend to put the spec steps as close as possible to the code they are referencing, so ideally these should be moved down a couple of lines.

Reply:
Yep, I had that before but it felt weird for some reason 😅 
Fixed now